### PR TITLE
[fix] Add cid to nav menu links

### PR DESCRIPTION
**Pull request by alex-deaconu**
Update cid when the company changes

Fix for issue #487.

@olegrise please review. Thanks!

### CODE REVIEW

File: dist/js/common-header.js [hunk 1]
```diff
@@ -5089,6 +5089,34 @@ function (loadFastpass, userState) {
   };
 }]);
 
+angular.module("risevision.common.header")
+.directive("linkCid", ["userState",
+function (userState) {
+  return {
+    link: function($scope, ele, attr){
+      var updateLinkCompanyId = function(companyId) {
+        var index = attr.href.indexOf("?cid=");
```
Comment (olegrise) [inline dist/js/common-header.js]: @alex-deaconu what if `cid` is not the first parameter? Could be `"?cid="` or `"&cid="`. Possible?

**Comment by alex-deaconu** [comment]
@olegrise updated the parsing as per your query. It should handle `?` and `&` correctly. Thanks!

**Comment by olegrise** [comment]
:+1:  @alex-deaconu I just realized that my comment was irrelevant to the way you use the directive. Sorry.

**Comment by alex-deaconu** [comment]
@olegrise no worries. The extra check doesn't hurt.

**Pull request merged**